docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+xbundle:
+  build: .
+  volumes:
+    - .:/src
+    - /tmp
```

Comment:
This is copied from LORE's docker-compose. It looks like you put it there originally during the original course import work, not sure why. In any case we can probably remove it here.